cyctl/cmd/apply.go

```diff
@@ -103,6 +126,8 @@ var applyCmd = &cobra.Command{
 			log.Fatal(err)
 		}
 
+		telemetry, _ := cmd.Flags().GetBool("disable-telemetry")
```

Comment:
Can you handle the error and rename the variable to `disableTelemetry`?
```suggestion
		disableTelemetry, err := cmd.Flags().GetBool("disable-telemetry")
		if err != nil {
			log.Fatal(err)
		}
```